app/index.html

```diff
@@ -14,6 +14,21 @@
 
     {{content-for 'head-footer'}}
   </head>
+  <script type="text/javascript">
```

Comment:
LOL... it was in the right place before... but when you have nested tags, they need to be indented 2 spaces deeper.

```html
<body>

  .... content ...

  <script type="text/javascript">
  </script>
</body>
```